### PR TITLE
v6.3.26 changelog

### PR DESCRIPTION
**Pull request by hzoo**

### CODE REVIEW

**Comment by amasad** [comment]
thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,20 @@ _Note: Gaps between patch versions are faulty, broken or test releases._
 
 See [CHANGELOG - 6to5](CHANGELOG-6to5.md) for the pre-4.0.0 version changelog.
 
+## 6.3.26
+
+* **Bug Fix**
+ * `babel-plugin-transform-es2015-parameters`: [#3191](https://github.com/babel/babel/pull/3191) Fix the order of arguements intitialization (fixes [T6809](http://phabricator.babeljs.io/T6809))
```
Comment (thejameskyle) [inline CHANGELOG.md]: s/arguements/arguments

Comment (thejameskyle) [inline CHANGELOG.md]: s/intitialization/initialization

**Comment by hzoo** [comment]
Fixed typos

**Pull request merged**